src/components/VersionList/VersionItem.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classes from 'classnames';
+import {Mutation} from 'react-apollo';
+import {FILE_DOWNLOAD_URL} from '../../state/mutations';
+import {Avatar, Icon, GridContainer} from 'kf-uikit';
+import TimeAgo from 'react-timeago';
+import CopyButton from '../CopyButton/CopyButton';
+import {
+  fileTypeDetail,
+  versionState,
+  formatFileSize,
+  downloadFile,
+} from '../../common/fileUtils';
+import SvgIcon from '../Icon/Icon';
+/**
+ * Displays single version item from the list
+ */
+const VersionItem = ({
+  className,
+  studyId,
+  fileId,
+  fileType,
+  versionNode,
+  index,
+}) => {
+  let versionItemClass = classes('FileVersionList--Element', className);
+  const stateColor = versionNode.state
+    ? versionState[versionNode.state].color
+    : 'bg-lightGrey';
+  const iconClass = classes(
+    'FileVersionElement--Icon',
+    'row-4',
+    'cell-1',
+    'lg:row-1',
+    'lg:cell-1',
+    stateColor,
+  );
+  return (
+    <Mutation mutation={FILE_DOWNLOAD_URL}>
+      {downloadFileMutation => (
+        <li key={versionNode.kfId} className={versionItemClass}>
+          <GridContainer collapsed="rows" className="p-4">
+            <div className={iconClass}>
+              <SvgIcon
+                kind={fileTypeDetail[fileType].icon}
+                width="14"
+                height="14"
+              />
+            </div>
+            {index === 0 && (
+              <div className="FileVersionElement--Tag row-4 cell-2 lg:row-2 lg:cell-1">
+                Latest
+              </div>
+            )}
+            <div className="flex row-1 cell-12 lg:cell-7">
+              <button
+                className="FileVersionElement--button"
+                onClick={e =>
+                  downloadFile(
+                    studyId,
+                    fileId,
+                    versionNode.kfId,
+                    downloadFileMutation,
+                  )
+                }
+              >
+                {versionNode.fileName}
+              </button>
+              <CopyButton
```

Comment:
The positioning here is a little confusing as it almost implies that you are copying the file name, not the kf id.
It would be helpful if we could move this down below the avatar with the actual kf_id displayed.
Maybe something like:
<img width="858" alt="Screen Shot 2019-06-06 at 10 11 59 AM" src="https://user-images.githubusercontent.com/2495894/59039835-8fe75680-8843-11e9-9a01-59f3b39d760a.png">